integration_tests/src/main/python/get_json_test.py

```diff
@@ -50,6 +50,19 @@ def test_get_json_object_quoted_index():
         f.get_json_object('jsonStr',r'''$['b']''').alias('sub_b')),
         conf={'spark.rapids.sql.expression.GetJsonObject': 'true'})
 
+def test_get_json_object_single_quotes():
```

Comment:
@revans2 Added `test_get_json_object_single_quotes`. This test would fail when `.allowSingleQuotes(false)`.